exp/views/responses.py

```diff
@@ -1955,7 +1955,6 @@ def post(self, request, *args, **kwargs):
         Downloads study video
         """
         match = self.request.GET.get("match", "")
-        orderby = self.get_ordering()
```

Comment:
this can't be deleted because the view's `orderby` is used, even though it's not used *here* in our code

Reply:
whoops, as discussed this is fine to delete - I thought it was defined on the class rather than in post